sources/packages/web/src/types/contracts/ConfigContract.ts

```diff
@@ -2,17 +2,15 @@ export interface GetConfig {
   auth: AuthConfig;
 }
 
+export type ClientIdType = "student" | "institute";
```

Comment:
Please consider changing it to a string enum.

Reply:
DONE